cowprotocol/cowswap
# Limit orders/receipt

# Summary

Implements #1338

This PR adds implementation of receipt modal

# To test
- go to limit-orders page
- create a new order or open order history tab
- click on some order in the table
- the new receipt modal should open

![Screenshot from 2022-11-24 18-12-49](https://user-images.githubusercontent.com/34926005/203838089-c4f62c75-1b65-45bc-bfd2-098d3a42d437.png)
